graph-permissions-schema.json

```diff
@@ -85,7 +85,12 @@
                 "schemeKeys": {
```

Comment:
Just out of curiosity... was the intention here to name this `schemeKeys` or `schemaKeys`? 🤔

Reply:
`schemeKeys` https://github.com/microsoftgraph/msgraph-metadata/blob/dm/permissionspec/specs/permissions.md#schemekeys